lib/config/presets/index.ts

```diff
@@ -266,7 +269,9 @@ export async function resolveConfigPresets(
   inputConfig: AllConfig,
   baseConfig?: RenovateConfig,
   _ignorePresets?: string[],
-  existingPresets: string[] = []
+  existingPresets: string[] = [],
+  shallowLog: shallowLogMode = 'off',
```

Comment:
From a preset point of view, I think it's shallow *resolving*, not shallow *logging*. The config/presets code resolves presets, it does not log them

Reply:
Yes, this flag that was added doesn't alter the function logic its purpose is to keep what is being done with only the extra logging of the inputConfig combined with the fetched config that is being extended externally by the user. This way we will save wasted function calls just for logging